views/start.erb

```diff
@@ -1,10 +1,10 @@
-<h1>Choose rock, paper, or scissors</h1>
-<p>
+<h1 style="text-align:center;">Choose rock, paper, or scissors</h1>
+<p style="text-align:center;">
```

Comment:
Idk html very well but is it possible to put all these links under one text align p tag set?
That's literally the only improvement I could think of adding to your code.